tests/unit/test_hole_in_one.py

```diff
@@ -75,30 +74,39 @@ def test_hole_in_two(tmp_path, clean, which_files, dpp, caplog):
         _check_all_harpy_files(analysis_output, sdds_file)
 
     # Run optics on the analysis output
-    optics_output = tmp_path / "optics_output"
-    hole_in_one_entrypoint(
-        beam=1,
-        optics=True,
-        accel="lhc",
-        year="2022",
-        model_dir=MODEL_DIR,
-        files=[analysis_output / sdds_file.name for sdds_file in sdds_files],
-        nonlinear=['rdt', 'crdt'],
-        outputdir=optics_output,
-    )
-
-    _check_linear_optics_files(optics_output, off_momentum=(which_files == "all"))
-    _check_nonlinear_optics_files(optics_output, "rdt")
-    _check_nonlinear_optics_files(optics_output, "crdt")
-
-    _check_caplog_for_rdt_warnings(caplog, to_be_found=(which_files == "all") and (dpp is None))
+    for dpp in (None, 0):
```

Comment:
Why not as a fixture anymore?

Reply:
I would prefer it as fixture, but then we run also the harpy part 6 times, which does not change. On the other hand, I think the optics-measurement part is the one that takes the longest anyway...

Reply:
The new tests actually add 20min to the total test times (extended tests seem to increase, 20min -> 30min, coverage test goes from 40min -> 58min). To be seen if we want this ....

Reply:
Big oof...